### DISCUSSION TITLE
WSLUtilities Unknown -  failed to synchronize all databases (invalid or corrupted database (PGP signature))

### DISCUSSION THREAD

**Discussion by call-1511** [sileshn/EndeavourOSWSL2#5] [General]
### Requirements

- [X] Yes
- [ ] No

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Version

Microsoft Windows [Version 10.0.19045.3086]

### Kernel Version

Linux version 5.15.90.1-microsoft-standard-WSL2 

### Repro Steps

 sudo pacman -Syu
[sudo] password for abcdef:
error: wslutilities: key "2D4C887EB08424F157151C493DD50AA7E055D853" is unknown
:: Import PGP key 2D4C887EB08424F157151C493DD50AA7E055D853? [Y/n] y
error: key "2D4C887EB08424F157151C493DD50AA7E055D853" could not be looked up remotely
:: Synchronizing package databases...
 endeavouros                                  27.6 KiB  9.79 KiB/s 00:03 [----------------------------------------] 100% core is up to date
 extra                                         8.3 MiB  1187 KiB/s 00:07 [----------------------------------------] 100% community is up to date
 multilib                                    140.7 KiB  41.4 KiB/s 00:03 [----------------------------------------] 100% wslutilities                               1153.0   B  1623   B/s 00:01 [----------------------------------------] 100%error: wslutilities: key "2D4C887EB08424F157151C493DD50AA7E055D853" is unknown
:: Import PGP key 2D4C887EB08424F157151C493DD50AA7E055D853? [Y/n] y
error: key "2D4C887EB08424F157151C493DD50AA7E055D853" could not be looked up remotely
error: failed to synchronize all databases (invalid or corrupted database (PGP signature))

### Expected Behavior

pacman should update the repositories and the relevant packages.

### Actual Behavior

 Import PGP key 2D4C887EB08424F157151C493DD50AA7E055D853? [Y/n] y
error: key "2D4C887EB08424F157151C493DD50AA7E055D853" could not be looked up remotely
error: failed to synchronize all databases (invalid or corrupted database (PGP signature))

**Comment by sileshn** [comment]
https://github.com/wslutilities/wslu/issues/278#issuecomment-1589870056

**Comment by call-1511** [comment]
Thanks @sileshn .

Following the comment and the specific [post]( https://github.com/wslutilities/wslu/issues/278#issuecomment-1589870056), I was able to resolve the issue. 

Appreciate the prompt response!
